spec/providers/coreIntegration/udpsocket.integration.src.js

```diff
@@ -27,14 +27,13 @@ module.exports = function(provider, setup) {
       return did.bind(null, markTask(name));
     };
 
-    var LOCALHOST = '127.0.0.1';
+    var LOCALHOST_V4 = '127.0.0.1';
     var checkSocketInfo = function(socketToCheck, port) {
       var getInfoTask = markTask('getInfo');
       socketToCheck.getInfo(function(state) {
-        expect(state).toEqual(jasmine.objectContaining({
-          'localAddress': LOCALHOST,
-          'localPort': port
-        }));
+        // On Chrome, this is "127.0.0.1".  On Firefox it's "localhost".
```

Comment:
@willscott It turns out this test doesn't work on node, because freedom-for-node is stuck on jasmine-1.14, and this test uses some jasmine-2.0 features (e.g. spy.and.callFake instead of spy.andCallFake).  So I don't know what it is on node ... and I'm not really sure how to proceed there, or with this pull request.  Suggestions welcome.